packages/fx-core/src/core/middleware/envInfoLoader.ts

```diff
@@ -101,10 +103,15 @@ export function EnvInfoLoaderMW(isMultiEnvEnabled: boolean): Middleware {
       }
 
       if (isV2()) {
-        //TODO core should not know the details of envInfo
-        ctx.envInfo = result.value.envInfo.config;
-        ctx.provisionOutputs = result.value.envInfo.profile;
-        ctx.envName = result.value.envInfo.envName;
+        const envInfo = result.value.envInfo;
+        const profile: Json = {};
+        for(const key of envInfo.profile.keys()) {
+          const map = envInfo.profile.get(key);
+          if(map){
```

Comment:
Looks like code style issue. `if(map)` => `if (map)`

@wenytang-ms would you please checking why the pipeline (Lint PR) doesn't work. From my understanding, it should fail here, right?

Reply:
sure, it's better rebase dev, then will trigger the pr format check

Reply:
@jayzhang could you help rebase the dev branch and the new pipeline (Lint PR) will be triggered to avoid the inconsistent code style checked-in? Thanks~